packages/models/proto/v2/eservice/eservice.proto

```diff
@@ -84,6 +89,20 @@ message EServiceDescriptorV2 {
   optional int64 archivedAt = 17;
   EServiceAttributesV2 attributes = 18;
   repeated DescriptorRejectionReasonV2 rejectionReasons = 19;
+  optional EServiceTemplateVersionRefV2 templateVersionRef = 20;
+}
+
+message TemplateInstanceInterfaceMetadataV2 {
+  string name = 1;
+  string email = 2;
+  string url = 3;
```

Comment:
same suggestion for renaming

Reply:
handled in this https://github.com/pagopa/interop-be-monorepo/pull/1620